build-tools/syncpack.config.cjs

```diff
@@ -105,6 +121,7 @@ module.exports = {
 				"@fluid-tools/version-tools",
 				"@fluidframework/build-tools",
 				"@fluidframework/bundle-size-tools",
+				"@fluidframework/build-tools-bin",
```

Comment:
This is how packages in the build-tools release group reference an older version of build-tools for their own builds.